tests/compiler/std/typedarray.json

```diff
@@ -1,7 +1,6 @@
 {
   "asc_flags": [
     "--runtime half",
-    "--explicitStart",
-    "--use ASC_RTRACE=1"
+    "--explicitStart"
```

Comment:
`std/typedarray` is one of the more interesting tests, but it's missing RTRACE now

Reply:
Oh, weird. Did not mean to do that.